conf/editor.go

```diff
@@ -475,7 +475,7 @@ func DatabaseMenu() {
 			Name:    "MySQL Version",
 			Type:    "menu",
 			Current: GetConfig().Database.Mysqlconfig.Version,
-			Default: "5.6",
+			Default: "5.7",
```

Comment:
Possibly a risky change for some existing users?

Reply:
Actually our default was always 5.7 - this changes it from an earlier mistake I made :P